pkg/vault/test/foundry/E2eSwap.t.sol

```diff
@@ -456,7 +498,13 @@ contract E2eSwapTest is BaseVaultTest {
         }
 
         if (testLocals.shouldTestSwapAmount) {
-            exactAmountOut = bound(exactAmountOut, minSwapAmountTokenB, maxAmountOut);
+            // If the liquidity is very small for one of the tokens and decimals are small too, the maxAmountOut may be
+            // smaller than minSwapAmount (usually 10^7), so just overwrite it.
+            if (minSwapAmountTokenB > maxAmountOut) {
+                exactAmountOut = bound(exactAmountOut, maxAmountOut, maxAmountOut);
```

Comment:
I did this because bound prints the number, so it's easier to debug, but I think it's better to remove it and just put the correct number, as you suggests.